packages/steps/src/bin/uploadArtifact.ts

```diff
@@ -0,0 +1,39 @@
+import path from 'path';
+
+import { createLogger } from '@expo/logger';
+import arg from 'arg';
+
+import { nullthrows } from '../utils/nullthrows.js';
+import { BuildArtifactType } from '../BuildArtifacts.js';
+import { BuildStepContext } from '../BuildStepContext.js';
+import { saveArtifactToTemporaryDirectoryAsync } from '../BuildTemporaryFiles.js';
+
+const args = arg({
+  '--type': String,
+});
+
+const artifactPathArg = nullthrows(
+  args._?.[0],
+  'upload-artifact must be run with the artifact path, e.g. "upload-artifact path/to/app.ipa"'
+);
+const artifactPath = path.resolve(process.cwd(), artifactPathArg);
+
+const logger = createLogger({
```

Comment:
won't this produce json logs? If this is spawned as an external process then logs from here would end up as serialized json inside other json? Am I missing sth?